i6_models/parts/conformer/convolution.py

```diff
@@ -0,0 +1,59 @@
+from torch import nn
+
+
+class ConformerConvolutionV1(nn.Module):
```

Comment:
Do we really want to put `V1` postfix on every single module now?

I think it makes more sense to introduce versions only once we really have multiple versions. For many of our modules, we might never get to that point.

Reply:
Yes, this was decided. This makes it more obvious that versioning is done per class individually, and we have a consistent pattern.

Reply:
To clarify: For every single module, we want to add this?

Reply:
@curufinwe 

I understood yes.

Reply:
yes i also understood the same.

Reply:
Ok I'm not sure this is not overkill. I would have sticked to not adding `V1` to anything, and only add `V2` once we really get to the point that we add a new incompatible version. Which should really be a rare case.